examples/react-signer/src/service/method/interactive/icrc49-call-canister-method.service.ts

```diff
@@ -96,7 +96,7 @@ class Icrc49CallCanisterMethodService extends InteractiveMethodService {
     )
     const delegation = DelegationIdentity.fromDelegation(sessionKey, chain)
 
-    const agent: Agent = new HttpAgent({
+    const agent: HttpAgent = HttpAgent.createSync({
```

Comment:
just in case, won't this be a problem with local development in future?

Reply:
It's a mocked signer. It isn't configured to work with local dfx anyways yet.